### DISCUSSION TITLE
./run.sh: line 23: ./Dynaframe: cannot execute: required file not found.

### DISCUSSION THREAD

**Discussion by Lakiman7** [Geektoolkit/Dynaframe3#236] [General]
I have tried with multiple versions of the OS on both the Raspberry Pi 3B+ and 4B, but the same error appears in the log: ./run.sh: line 23: ./Dynaframe: cannot execute: required file not found.

**Comment by Lakiman7** [comment]
Can someone please help? 
I see that several people have the same problem...

**Reply by Geektoolkit** [reply]
This repository at this point is basically if someone needs sample code or a starting point. This only works on raspbian versions which are long out of support. The Dynaframe pro version is one which me and a few team members work to keep running, and support. And even that is being hit by the same issues that took this down...raspbian updates that removed features or broke things so badly that it constantly required a rewrite

**Reply by Lakiman7** [reply]
Thank you for your response.
